exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Op.kt

```diff
@@ -552,7 +559,7 @@ class RegexpOp<T : String?>(
 /**
  * Represents an SQL operator that checks if [query] returns at least one row.
  */
-class exists(
+class Exists(
```

Comment:
Potential breaking change introduced here. If we don't want this, I can revert this change and suppress the `detekt` warning.